src/screens/Login/index.tsx

```diff
@@ -0,0 +1,127 @@
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+
+import { AxiosError } from 'axios';
+
+import AuthAdapter from 'adapters/authAdapter';
+import logo from 'assets/images/logo.svg';
+import Button from 'components/Button';
+import Input from 'components/Input';
+import { setToken } from 'helpers/userToken';
+import { isEmailValid, isPasswordValid } from 'helpers/validators';
+
+import { PASSWORD_MIN_LENGTH } from '../../constants';
+
+function LoginScreen() {
+  const navigate = useNavigate();
+  const { t } = useTranslation('translation');
+
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [errors, setErrors] = useState<string[]>([]);
+  const [formSubmitted, setFormSubmitted] = useState(false);
+
+  const handleEmailChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setEmail(e.target.value);
+  };
+
+  const handlePasswordChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setPassword(e.target.value);
+  };
+
+  const performLogin = async () => {
+    try {
+      const response = await AuthAdapter.login({ email: email, password: password });
+
+      const {
+        attributes: { access_token: accessToken, refresh_token: refreshToken },
+      } = await response.data;
+
+      /* eslint-disable camelcase */
+      setToken({ access_token: accessToken, refresh_token: refreshToken });
+      /* eslint-enable camelcase */
+
+      navigate('/');
+    } catch (error) {
+      let errorMessage = 'There was a problem receiving a response from the server';
```

Comment:
Should we add this base error message to the translation file?

Reply:
Yup, forgot that one :)